python/pyspark/ml/deepspeed/deepspeed_distributor.py

```diff
@@ -35,11 +35,11 @@ class DeepspeedTorchDistributor(TorchDistributor):
 
     def __init__(
         self,
-        num_gpus: int = 1,
+        numGpus: int = 1,
```

Comment:
cc @rithwik-db on the naming. Is the name like `ngpus`, `local` better? or the current ones are better?

Reply:
I think it would be best to stay as similar to the Deepspeed cli as possible; [Deepspeed](https://www.deepspeed.ai/getting-started/) uses `--num_gpus` so I think `numGpus` would be closer to that API than `ngpus`